bazaarvoice/emodb
# Fix cleanup tables sh script

## What Are We Doing Here?

We have cleanup script which should allow us to define table prefix for finding all prefixed tables and remove them.
At the moment of fix there were few mistakes like wrong api method used, tables were looked for not by prefix but by substring.  Additionally, I have removed possibility to supply more than one prefix for searching and removing tables as even though we have prefixes we use for testing purposes I believe we should carefully check list of tables that were gathered by script for removal.

## How to Test and Verify

1. Check out this PR
2. Creat test table(s) (Make sure that there is only your table(s) has/have specified table name prefix)
3. Update cleanup_tables.sh with table prefix which corresponds to table created for test purposes
4. Run ./quality/integration/src/test/resources/cleanup_tables.sh script with --host -key (--dry-run and without --dry-run)
5. Verify that table just found by prefix and no removal has been performed (when there is --dry-run option is used)
6. Verify that test table has been found and removed when there is no --dry-run option is used

## Risk

### Level 

`Low`

### Required Testing

`Manual`
### Risk Summary
At the moment script doesn't work so there only thing is that it still not working. 
There is no dependancy on this script from CI/CD system or by any other systems/activities we do because of this, fix is not expended to have potential harm.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
